stateproofs/stateProofVerifier.go

```diff
@@ -0,0 +1,33 @@
+package stateproofverification
```

Comment:
It seems weird that the package name is not the same as the directory name, could you rename the package or the directory?